.github/CONTRIBUTING.md

```diff
@@ -10,10 +10,7 @@ This site is built with [Primer React components][components] and [x0].
 To start up your local dev environment, run:
 
 1. `npm install` to install all the Node dependencies
-2. `npm run dev` to start the dev server
-3. `npm run build` to build the static site
-4. `npm start` to serve the static site
-
+2. `npm start` to run the app in development mode
```

Comment:
```suggestion
2. `npm start` to run the app in development mode

Additionally, you can run `npm run now-build` to build the static site, and `npm run now-start` to run the static site on your local environment. This can be helpful for debugging build issues.
```

Reply:
Hey @gtsiolis this looks great, just made one suggestion!

Reply:
Also! Yes, that would be awesome to have the lint script documented here!

Reply:
Thanks @emplums! I've applied the suggestion as is. 🏓